src/paragraphcommand.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module paragraph/paragraphcommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command/command';
+
+/**
+ * The paragraph command.
+ *
+ * @extends module:core/command/command~Command
+ */
+export default class ParagraphCommand extends Command {
+	/**
+	 * Creates an instance of the command.
+	 *
+	 * @param {module:core/editor/editor~Editor} editor Editor instance.
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		const t = editor.t;
+
+		/**
+		 * Value of the command, indicating whether it is applied in the context
+		 * of current {@link module:engine/model/document~Document#selection selection}.
+		 *
+		 * @readonly
+		 * @observable
+		 * @member {Boolean}
+		 */
+		this.set( 'value', false );
+
+		/**
+		 * User-readable title of the command, for use in the UI.
+		 *
+		 * @readonly
+		 * @member {String}
+		 */
+		this.title = t( 'Paragraph' );
```

Comment:
I don't think it makes sense to provide the title here. If someone wants to use this command in his/her UI that person should provide the title there.